tests/general/test.sh

```diff
@@ -17,17 +17,15 @@ rolesInstallAnsible() {
     local ansible_pkg
     local pkg_cmd
     if rlIsFedora || (rlIsRHELLike ">=8.6" && [ "$ANSIBLE_VER" != "2.9" ]); then
-        pkg_cmd="dnf"
-        ansible_pkg="ansible-core-$ANSIBLE_VER"
+        rlRun "dnf install python3.12-pip -y"
+        rlRun "python3.12 -m pip install ansible-core==$ANSIBLE_VER"
```

Comment:
```suggestion
        rlRun "python3.12 -m pip install ansible-core==$ANSIBLE_VER.*"
```
That is - if you do `pip install ansible-core==2.17` I don't know if it will install the version that matches exactly, or if it will install the latest version of 2.17.x.  Using `pip install ansible-core==2.17.*` will ensure that you get the latest version of 2.17.x.